.github/workflows/ci.yml

```diff
@@ -0,0 +1,45 @@
+name: CI
+on:
+  pull_request:
+  push:
+    branches:
+      - master
+    tags:
+      - '*--*'
```

Comment:
What's special about this tag?

Reply:
We expect the tag to be `<artifact_id suffix>--<version>`